src/actions/search.js

```diff
@@ -0,0 +1,31 @@
+const GET_SEARCH_REQUESTED = 'GET_SEARCH_REQUESTED';
+const GET_SEARCH_COMPLETED = 'GET_SEARCH_COMPLETED';
+const GET_SEARCH_FAILED = 'GET_SEARCH_FAILED';
+
+const getSearchRequested = (search = '') => ({
+  type: GET_SEARCH_REQUESTED,
+  queryStr: search,
```

Comment:
Action creators should wrap the payload inside an object:

```
const getSearchRequested = (gif) => (
  type: GET_SEARCH_REQUESTED,
  payload: { queryStr: search },
);
```
This is in order to have a standard way of transporting payloads, and
comply with the Flux standard.